testcafe/tests/photos/photos_start_upload_photos.js

```diff
@@ -3,17 +3,34 @@ import { TESTCAFE_PHOTOS_URL } from '../helpers/utils'
 import Page from '../pages/photos-model'
 import { DATA_PATH, IMG0, IMG1, IMG2, IMG3, IMG4 } from '../helpers/data'
 
+import VisualReview from '../../VisualReview-node-client/index'
+
+const vr = new VisualReview({
+  //  debug: true,
+  protocol: 'https',
+  hostname: 'visualreview.cozycloud.cc',
+  projectName: `PHOTOS on ${TESTCAFE_PHOTOS_URL}`,
```

Comment:
Shouldn't the project name be something more unique, like just the app name?

Reply:
Yes, I change it back to slug, otherwise, we'll need to validate on each server, and we dont want that!